todo_app/data/trello_items.py

```diff
@@ -0,0 +1,67 @@
+import requests
+import os
+
+url_main_cards = 'https://api.trello.com/1/cards/'
+Board_ID = os.environ.get('BOARD_ID')
+base_query_parameters = {'key': os.environ.get('key'),
+                                'token': os.environ.get('token')}
+
+def fetch_all_cards():
+    r_json = json_of_all_cards()
+    return r_json
+
+def create_new_card(title):
+    url = url_main_cards
+    data_todo_list = (define_lists_in_board())[0]
+
+    query = base_query_parameters
```

Comment:
Please can you add these two helper functions and implement them everywhere appropriate?


```suggestion
def get_auth_params():
    return { 'key': environ.get('TRELLO_API_KEY'), 'token': environ.get('TRELLO_API_SECRET') }

def build_params(params = {}):
    full_params = get_auth_params()
    full_params.update(params)
    return full_params
```